### DISCUSSION TITLE
Feature request: Log server configurations on startup

### DISCUSSION THREAD

**Discussion by grizzlybearer** [openfga/community#289] [Ideas]
It would be great if we have a way of checking the OpenFGA server configs that were loaded, for instance, in the server logs at startup time.
This is particularly useful to understand the overridden configs.

Spun up from discord discussion: https://discord.com/channels/759188666072825867/920359187949707305/1224388148361035876

**Comment by aaguiarz** [comment]
This was implemented :)
